docs/END_USER_DOCUMENT.md

```diff
@@ -750,3 +751,38 @@ AsyncBooksAppClient asyncClient = MoccaClient.Builder
         .client(executorClient)
         .build(AsyncBooksAppClient.class);
 ```
+## 8. Request Validation
+Mocca now supports validation of request parameters using a standard bean validation 2.0 implementation like hibernate.
+Please refer to [this site](https://beanvalidation.org/2.0-jsr380/) for information on bean validation. 
+
+**Important Note:** Mocca supports Bean Validation 2.0, not Jakarta Bean Validation 2.0 which has repackaged the api 
+classes from `java.validation` to `jakarta.validation`.
+
+### 8.1 Adding Request Validation to your Application
```

Comment:
Can we add an example of a client interface with Bean Validation annotations?

Reply:
sure